utils/appliance/__init__.py

```diff
@@ -89,6 +89,8 @@ def run_commands(self, commands, autoreturn=True, timeout=20, channel=None):
             try:
                 while True:
                     result += channel.recv(1)
+                    if result is 'Press any key to continue':
```

Comment:
@lcouzens: this will work in this certain case but please use == to check that objects have equal values. "is" is for other things.